.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: bash
+sudo: required
+dist: trusty
+
+before_install:
+  - sudo apt-get update -q
+  - sudo apt-get install cabal-install -y
+  - cabal update
+  - cabal install shellcheck
+script:
+  - ~/.cabal/bin/shellcheck concurrent.lib.sh
+  - ~/.cabal/bin/shellcheck demo.sh
```

Comment:
Let's merge these together into:

```
  - ~/.cabal/bin/shellcheck *.sh
```

This should provide the following benefits:

- No need to update this file if we add more scripts.
- If both scripts contain warnings, all messages will be output instead of Travis exiting after the first failed command.